### PR TITLE
Feature/move functions

### PR DESCRIPTION
**Pull request by monicagerber**
**Summary:** I moved functions to the R/ folder in VISCtemplates, documented functions, updated the writing (based on the BAMA memo template [here](https://github.com/FredHutch/VISC-Assay-Methods/blob/master/Assay_Memo_Templates/serum_BAMA.md), and some minor formatting changes.

**Details and some questions**
1. `install_load_cran_packages()`: should we still assign this to `package.check`? 
2. `visc_ref()` and `visc_clearpage()`: what do you think about renaming these to `insert_ref()` and `insert_break()`?
3. `opts_chunk$set`: removed `tidy=TRUE` because this re-formats the _display_ of code chunks, not the actual code written in the R Markdown file, and we don't display code chunks in PT reports. Removed `fig.height` and `fig.width` because `7` is already the default.
4. Changed the data package example to HTTPS so that it's simpler and could run without changing the SSH path.

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # Generated by roxygen2: do not edit by hand
 
 export(create_visc_project)
```
Comment (wfulp) [inline NAMESPACE]: When I rebuild I get a `export(check_pandoc_version)` line added to the namespace.

File: R/report_template.R [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+         'latex', 'pandoc')
+
+}
+
+#' Set kable warnings based on output type
+#'
+#' This can be used to set the `warning` option in R Markdown code chunks to
+#' remove warnings created by knitr::kable() when knitting to a Word document.
+#'
+#' @param output_type
```
Comment (wfulp) [inline R/report_template.R]:
```suggestion
#' @param output_type character string of document output type 
```

File: R/report_template.R [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+set_kable_warnings <- function(output_type) {
+  ifelse(output_type == 'latex', TRUE, FALSE)
+}
+
+#' Set pandoc markup
+#'
+#' Use this for conditionally formatting output when knitting both
+#' PDF (Latex) and Word document reports.
+#'
+#' @param output_type
```
Comment (wfulp) [inline R/report_template.R]:
```suggestion
#' @param output_type character string of document output type 
```

File: NAMESPACE (hunk 1 above)
Comment (monicagerber) [reply NAMESPACE:4]: Forgot to commit this change! It's updated now.

**Comment by wfulp** [comment]
@monicagerber I don't think `visc_clearpage` made it into the report_template.R file.

```
visc_clearpage <- function() {
  ifelse(knitr::opts_knit$get('rmarkdown.pandoc.to') == 'latex', 
          '\\clearpage', 
         '#####')
} 
```

**Pull request merged**